server/services/locationService.test.ts

```diff
@@ -0,0 +1,75 @@
+import nock from 'nock'
+import config from '../config'
+import LocationService from './locationService'
+import Location from '../data_models/location'
+
+// BE CAREFUL THIS IS MOCKING THE CORRECT THING. NOT: jest.mock('../data/restClient')
+jest.mock('../data/hmppsAuthClient')
```

Comment:
Is this being used by the LocationService?

Reply:
Turns out, no!